src/Hedgehog/Gen.fs

```diff
@@ -69,20 +69,32 @@ module Gen =
     let map (f : 'a -> 'b) (g : Gen<'a>) : Gen<'b> =
         mapTree (Tree.map f) g
 
+#if FABLE_COMPILER
+    [<CompiledName("Map2")>]
+#else
     [<CompiledName("Map")>]
+#endif    
```

Comment:
We don't actually need `CompiledName` in Fable.

```suggestion
#if !FABLE_COMPILER
    [<CompiledName("Map")>]
#endif    
```

But you can keep them for consistency with the .NET version.

Reply:
I removed all the CompiledNames for FABLE_COMPILER.